core/common/src/main/scala/org/typelevel/otel4s/context/propagation/TextMapPropagator.scala

```diff
@@ -30,10 +31,13 @@ import cats.Applicative
   * interceptors. On the client side, values are injected into the carriers,
   * while on the server side, values are extracted from them.
   *
-  * @tparam F
-  *   the higher-kinded type of a polymorphic effect
+  * @tparam Ctx
+  *   the context to use to extra or inject data
   */
-trait TextMapPropagator[F[_], Ctx] {
+trait TextMapPropagator[Ctx] {
+
+  /** The list of propagation fields. */
+  def fields: List[String]
```

Comment:
similar to `TextMapGetter#keys`, I think this should return `Iterable[String]`

Reply:
Good point. It should be `Iterable`.